app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitItemAdapter.java

```diff
@@ -20,7 +20,8 @@
 import java.util.Locale;
 
 public class HabitItemAdapter extends RecyclerView.Adapter<HabitItemAdapter.HabitViewHolder> {
-    private static final String DATE_PATTERN = "EEE, MMM d, yyyy";
+    //private static final String DATE_PATTERN = "EEE, MMM d, yyyy";
```

Comment:
I wasn't sure if we needed the commented-out line anymore, so I left it to be safe. I assumed maybe later on when we add what days habits will take place on it will be good to change it back. I commented it out just to keep a consistent format with adding a habit to the habit list, but I can remove it for my next pull request.

Reply:
Yeah, personally I'd just remove it. If there's no need for it now, no point in having it in there. Someone can easily just go in and change the data patter, that's why we made it a constant in the first place so this is easy.